app/templates/components/application/site-search/group/item.hbs

```diff
@@ -0,0 +1,18 @@
+<li class="media col-sm-12">
+  <a class="search--result-link" href="{{link}}" {{action "transitionTo" link}}>
+    <div class="media-left col-sm-2 no-padding-left">
+      {{#if (eq group "users")}}
```

Comment:
Better approach here might be to use `model-type` helper on `item` and see if it `=== user`